islandora-interest-groups/Islandora-DevOps-Interest-Group
# Issue 36

* Other Relevant Links (Google Groups discussion, related pull requests, Release pull requests, etc.)
https://github.com/islandora-interest-groups/Islandora-DevOps-Interest-Group/issues/36

# What does this Pull Request do?

Fixes issues outlined in https://github.com/islandora-interest-groups/Islandora-DevOps-Interest-Group/issues/36

# What's new?
Replaces jwplayer with videojs.
Does not place skel with "Fedora" user since it causes installer to prompt overwrite. 
Adds variable for resolvable hostname for adore-djatoka url. 
Adds symlink for compatibility with basic-solr-config.

# How should this be tested?

Run through the relevant sections of the deployment guide to see if anything fails. It might be prudent to run through the whole doc again. 


# Additional Notes:

# Interested parties
@bdgregg
